numpy/distutils/msvc9compiler.py

```diff
@@ -0,0 +1,22 @@
+import os
+from distutils.msvccompiler import *
```

Comment:
@charris msvc9compiler.py serves as decorator for several methods in ```distutils.msvc9compiler.MSVCCompiler```. But distutils.msvc9compiler.py contains not only ```MSVCCompiler``` class. ```<python>\...\distutils\msvc9compiler.py``` also contains another classes, functions, variables. ```import *``` is allowed to use these objects from ```<numpy>\distutils\msvc9compiler.py```.